src/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -198,9 +201,17 @@ private static string ReadHeaderFile(string? headerFile)
 
             """;
 
-            return !string.IsNullOrEmpty(headerFile) ?
+            string header = !string.IsNullOrEmpty(headerFile) ?
                 File.ReadAllText(headerFile) :
                 defaultFileHeader;
+
+#if NET
+            header = header.ReplaceLineEndings();
+#else
+            header = Regex.Replace(header, @"\r\n|\n\r|\n|\r", Environment.NewLine);
```

Comment:
I'd be surprised if `\n\r` is ever found in the wild. I think `@"\r\n|\r|\n"` is enough, it would cover that edge case.

Reply:
I saw others doing this kind of replacement so I thought it doesn't harm. This code path is .NET Framework only so I'm less concerned.